### PR TITLE
Add option to clear playlist

### PR DESCRIPTION
**Pull request by arshubham**
Stops playing current file as per the behavior of vlc an celluloid. Not so sure about if I should remove `current-video` from gsettings.

### CODE REVIEW

File: src/Widgets/Player/PlaylistPopover.vala [hunk 1]
```diff
@@ -37,6 +37,9 @@ public class Audience.Widgets.PlaylistPopover : Gtk.Popover {
         dvd = new Gtk.Button.from_icon_name ("media-optical-symbolic", Gtk.IconSize.BUTTON);
         dvd.set_tooltip_text (_("Play from Disc"));
 
+        var clear_playlist_button = new Gtk.Button.from_icon_name ("edit-clear", Gtk.IconSize.BUTTON);
```
[the remark this answers is not in the document]
Comment (arshubham) [reply src/Widgets/Player/PlaylistPopover.vala]: edit-delete-symbolic is much better here. edit-clear-symbolic was not descriptive enough here.

[the remark this answers is not in the document]
Comment (ryonakano) [reply src/Widgets/Player/PlaylistPopover.vala]: Yeah I think so too.

**Comment by arshubham** [comment]
@ryonakano @elementary/ux  Any opinion on if I should `current-video` from gsettings?

**Comment by ryonakano** [comment]
@arshubham You mean whether to clear the value of `current-video` when clearing a playlist or not?

**Comment by arshubham** [comment]
Yes

**Comment by ryonakano** [comment]
I personally think it's OK with this behavior that the value of  `current-video` is reset when clearing a playlist because it a user clears their playlist accidentally, their video files are not deleted or changed―just the progress is lost―and I don't think it's a critical thing.

But I'd also like to hear opinions from @elementary/ux.

**Pull request merged**